docs/userGuide/includingContents.md

```diff
@@ -25,8 +25,11 @@
 
     - `inline` (optional): make the included result an inline element. (wrapped in `<span>` tag)
 
+    - `if-exists` (optional): include the document only if it exists.
+
     Examples:
     ```html
+    <include src="exercises.md" if-exists/>

```

Comment:
Put it below all the other existing examples (after `<include src="../index.html" />`); it is not a typical use case so it shouldn't be at the top.